sap/5.2/modules/ROOT/pages/sap-connector-troubleshooting.adoc

```diff
@@ -180,62 +182,66 @@ image::sap-error-jco-classloader-fix.png[SAP Error JCo Classloader Fix]
 
 ==== Solution
 
 Ensure that the following requirements are met:
 
 * 64-bit JCo is required on a JVM that runs in 64-bit mode, and 32-bit JCo is required on a JVM that runs in 32-bit mode.
-* On Microsoft® Windows®, JCo requires the Microsoft Visual Studio 2005 C/C++ runtime libraries.
+* On Microsoft Windows, JCo requires the Microsoft Visual Studio 2005 C/C++ runtime libraries.
 * Both the `sapjco.jar`, and one of `sapjco3.dll` or `sapjco3.so` or `sapjco3.jnilib` must be from the *same JCo package*.
-+
-[NOTE]
-====
-To check the versions of the JCo libraries that you are using, do one of the following:
-
-* From the UI (Windows):
-	. Navigate to the directory where the sapjco3.jar file is located.
-	. Right-click the `sapjco3.jar` file.
-	. Select *Open With* from the context menu.
-	. Click *Java 2 Platform Standard Edition* binary.
-	. Verify the information shown in the JCo dialog that displays.
 
-* From a console:
-	. Open a terminal console.
-	. Navigate to the directory where the `sapjco3.jar` file is located.
-	. Execute the command `java -jar sapjco3.jar -version`.
-	. Verify the information shown in the JCo dialog that displays.
+To check the versions of the JCo libraries that you are using, do one of the following:
 
-====
+* On Windows (in the UI):
+. Navigate to the directory where the `sapjco3.jar` file is located.
+. Right-click the `sapjco3.jar` file.
+. Select *Open With* from the context menu.
+. Click *Java 2 Platform Standard Edition* binary.
+. Verify the information shown in the JCo dialog that displays.
+* On Mac:
+. Open a terminal window.
+. Navigate to the directory where the `sapjco3.jar` file is located.
+. Execute the command `java -jar sapjco3.jar -version`.
+. Verify the information shown in the JCo dialog that displays.
 
 [[error-jco-renaming-conflicts]]
 === JCo Renaming Conflicts
 
+==== Problem
+
+You receive the following error:
+
+[source,text,linenums]
 ----
 java.lang.ExceptionInInitializerError: Illegal JCo archive "sapjco3-3.0.11.jar". It is not allowed to rename or repackage the original archive "sapjco3.jar"
 ----
 
-==== Problem
-
 You cannot rename any of the SAP JCo library files in JCo 3.0.11 or later as they won't be recognized by JCo.
 
 ==== Solution
 
-If you are using *Apache Maven*, configure the *maven-dependency-plugin* with the attribute `<stripVersion>true</stripVersion>`. By doing this, all version numbers of the dependent libraries will be stripped when copying the JCo artifacts.
+If you are using *Apache Maven*, configure the *maven-dependency-plugin* with the attribute `<stripVersion>true</stripVersion>`. When you do this, all version numbers of the dependent libraries are stripped when copying the JCo artifacts.
 
 Further information is available externally at the http://maven.apache.org/plugins/maven-dependency-plugin/usage.html[Apache Maven Dependency Plugin].
 
 
 [[error-message-not-a-sap-object]]
-=== Message Not a SAP Object
+=== Message Not an SAP Object
+
+==== Problem
```

Comment:
This is a perfect example of what I suggest for troubleshooting subtopic titles: the actual message, issue, symptom. No need to descend another level for Problem and Solution.